schema/dataKeys.json

```diff
@@ -0,0 +1,352 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "fbc": {
+      "type": "object",
+      "properties": {
+        "request": {
+          "type": "string",
+          "description": "The internal pipline name to handle requests e.g. iib"
+        },
+        "fbcPublishingCredentials": {
+          "type": "string",
+          "description": ""
+        },
+        "buildTags": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "addArches": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "hotfix": {
+          "type": "boolean",
+          "description": "Indicates if the build is a hotfix"
+        },
+        "preGA": {
+          "type": "boolean",
+          "description": "Indicates if the build is a preGA"
+        },
+        "stagedIndex": {
+          "type": "boolean",
+          "description": "Indicates if the build is a staged index"
+        },
+        "productName": {
+          "type": "string",
+          "description": "The product name e.g. Red Hat Product "
```

Comment:
this string is used to construct the preGA tag name. It should be a plain lowercase string without whitespaces.

Reply:
fixed: Added a regex `"pattern": "^[a-z]+$"` and changed the example to exampleproduct